pkg/storage/chunk/chunk_store.go

```diff
@@ -219,7 +219,7 @@ func (c *store) GetChunkRefs(ctx context.Context, userID string, from, through m
 }
 
 // LabelValuesForMetricName retrieves all label values for a single label name and metric name.
-func (c *baseStore) LabelValuesForMetricName(ctx context.Context, userID string, from, through model.Time, metricName, labelName string) ([]string, error) {
+func (c *baseStore) LabelValuesForMetricName(ctx context.Context, userID string, from, through model.Time, metricName, labelName string, matchers ...*labels.Matcher) ([]string, error) {
```

Comment:
Do we need to make a change here ? Not sure if there's a reason for changing either the body or the signature of that function.

Reply:
Might be for the interface ?

Reply:
Yeah I guess it is.